src/components/AppDisplay.tsx

```diff
@@ -134,7 +123,12 @@ export default function AppDisplay({ children, data, GLink }: AppDisplayProps) {
     URL.revokeObjectURL(url);
   }, [appObject]);
 
-  const HEIGHT = 320;
+  useEffect(() => {
+    const ResultNode = require('./nodes/ResultNode').default;
+    setResultNodeTypes({
+      default: ResultNode,
+    });
+  }, []);
```

Comment:
What is the point of this compared to having the result node types declared outside of the component?

Reply:
@JeffDotPng Importing ResultNode component outside of this component throws an error in SSR in SSG. So to make sure that its only imported in client side I moved the import statement in useEffect hook.